deploy/crds/openliberty.io_openlibertyapplications_cr.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: openliberty.io/v1beta1
 kind: OpenLibertyApplication
 metadata:
   name: demo-app
+  namespace: default
```

Comment:
We probably should remove this from all example CRs to make it use current oc project

Reply:
Removed namespaces from all CRs.